spec.md

```diff
@@ -308,6 +308,23 @@ separated by a space. For example:
 class MyException extends \RuntimeException {}
 ```
 
+When accessing a class member immeditely after instantiating a new class, the instantiation SHOULD NOT be wrapped in 
```

Comment:
There's a small typo here, should be "immediately".

Reply:
Sigh.  Since it's already merged, can you send a tiny PR?  (I'm about to get on a plane.)